### ISSUE TITLE
How to preprosess noisy mesh and generate denoisy mesh

### ISSUE THREAD

**Issue by zhuzhiwei99** [Jia1018/ResGEM-Multi-scale-Graph-Embedding-Network-for-Residual-Mesh-Denoising#3]
How to preprosess the noisy mesh to get the input datas
![image](https://github.com/user-attachments/assets/3558f4e6-6188-4e7b-9b43-35076880d4e8)

I run the cmd `python infer.py --testset ./data/test --data_type Synthetic --graph_type DENSE_FACE --resume ./ckpt/Synthetic_nn.pth --save_dir ./data/results`
And I get the following result. What's the meaning of the result? Are they denoisy normals? How can I use these results to get denoisy mesh.
![image](https://github.com/user-attachments/assets/1cc3cb76-d98b-4831-95ef-15c772b80e03)

Hope author can provide necessary codes to get full results. thanks!

**Comment by Jia1018** [comment]
Yes, the results are the denoised normals of these models. The data preprocessing process is implemented in C++ code using the easy3D toolkit. The compilation and execution of the code are relatively complex, so the code is not provided. If you need the complete denoising results on the public dataset, you can download them directly from the following link:  https://drive.google.com/drive/folders/1cV3offz01vJhnpo8ECGSK-OYQKzT3OXJ?usp=drive_link. I hope this information proves helpful in your research, and please feel free to reach out if you have any further questions or require additional assistance.
